base/rtlib/RTLIB.jl

```diff
@@ -0,0 +1,129 @@
+module RTLIB
+
+
+# We would like to use `@ccallable` here,
+# but building the sysimage fails, so we use a bootstrapped version.
+function register(f, rtype, argt, name)
```

Comment:
@vtjnash I was trying to use `@ccallable` here but I was getting failures during sysimage building.